jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/profiles/ProfileReader.kt

```diff
@@ -37,6 +38,7 @@ private fun validateProfile(profile: Profile, allProfiles: Map<String, Profile>)
         profile.propertyExists(ProfileProperty.ROLE_ARN) -> validateAssumeRoleProfile(profile, allProfiles)
         profile.propertyExists(ProfileProperty.AWS_SESSION_TOKEN) -> validateStaticSessionProfile(profile)
         profile.propertyExists(ProfileProperty.AWS_ACCESS_KEY_ID) -> validateBasicProfile(profile)
+        profile.propertyExists(PROFILE_SSO_SESSION_PROPERTY) -> validateProfileSsoSession(profile)
```

Comment:
we also need to address the case where we have `sso-session`, but no associated profile

Reply:
Added this test case